Source/Core/DolphinWX/GameListCtrl.h

```diff
@@ -90,13 +95,22 @@ class GameListCtrl : public wxListCtrl
   static bool MultiCompressCB(const std::string& text, float percent, void* arg);
   static bool WiiCompressWarning();
 
-  std::vector<std::unique_ptr<GameListItem>> m_ISOFiles;
-  struct {
+  struct
+  {
     std::vector<int> flag;
     std::vector<int> platform;
     std::vector<int> utility_banner;
   } m_image_indexes;
 
+  // Actual backing GameListItems are maintained in a background thread and cached to file
+  static constexpr u32 CACHE_REVISION = 0;
```

Comment:
Do we want (have to?) keep the "last updated in PR" comment alive here, to make sure _something_ conflicts if two PRs change it at the same time? Unlikely, but never say never...